Added Create Category functionality

AyushTripathi-sketch opened the pull request:
Solved issue #103 

Category List Screen Before: 

![1st](https://user-images.githubusercontent.com/59537283/197812448-3a97211e-e7ba-48bb-8e3a-d22bd07ead05.jpg)

Category Create Screen:

![2nd](https://user-images.githubusercontent.com/59537283/197812547-3c3b3830-af7d-4960-af35-f9f97356f45a.jpg)

Category Screen After:

![3rd](https://user-images.githubusercontent.com/59537283/197812617-06f2707b-4c04-41f2-a4fd-b8320d9adf35.jpg)

roopeshsn commented:
Hi, @AyushTripathi-sketch Could you add a screen-recorded video of your work? Also, could you summarize what changes you have done? Have you formatted the code using prettier?

AyushTripathi-sketch commented:
Yes, I can scree record my work. I have created a screen for creating category and updated the category action and reducers for deleting, updating and creating category. In the backend I have made a category controller for different for the same actions and added a admin/createcategory route. I have also formatted the code using Prettier..
